AppDashboard/dashboard.py

```diff
@@ -1157,7 +1279,10 @@ def get(self):
   ('/logs/(.+)', LogServicePage),
   ('/gather-logs', LogDownloader),
   ('/groomer', RunGroomer),
-  ('/change-password', ChangePasswordPage)
+  ('/change-password', ChangePasswordPage),
+  ('/ajax/render/panel', AjaxRenderPanel),
+  ('/ajax/save/layout', AjaxSaveLayoutSettings),
+  ('/ajax/reset/layout', AjaxResetLayoutSettings)
```

Comment:
URLs should usually be constructed in increasing specificity. In this case, `/ajax/layout/reset` and `/ajax/layout/save` would make more sense.